Debian/bin/lib/Cyrus/Docker/Command/test.pm

```diff
@@ -0,0 +1,59 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::test;
+use Cyrus::Docker -command;
+
+use Path::Tiny;
+use Process::Status;
+
+sub execute ($self, $opt, $args) {
+  unless (-e '/run/rsyslogd.pid') {
+    system('/usr/sbin/rsyslogd');
+    Process::Status->assert_ok('starting rsyslog');
+  }
+
+  my $root = "/srv/cyrus-imapd/cassandane";
+  chdir $root or die "can't chdir to $root: $!";
+
+  unless (-e "cassandane.ini") {
+    system(qw(cp -af cassandane.ini.dockertests cassandane.ini));
+    Process::Status->assert_ok('copying cassandane.ini.dockertests to cassandane.ini');
```

Comment:
Actually all of these have "about to do text". Could be confusing if they succeed then something else goes sideways

Reply:
(We talked out of band, these are only printed if the thing fails. Okay!